user/waydroid/patches/apparmor-config.patch

```diff
@@ -0,0 +1,9 @@
+--- a/data/configs/config_3	2024-08-07 18:01:14.000000000 -0300
++++ b/data/configs/config_3	2024-09-07 14:21:28.466488580 -0300
+@@ -1,6 +1,5 @@
+ lxc.uts.name = waydroid
+ 
+-lxc.apparmor.profile = unconfined
```

Comment:
just out of curiosity, what error does lxc give when it's built without apparmor support but this config is there?

Reply:
it just says that LXC was build without apparmor support and exits. https://github.com/lxc/lxc/blob/bad6d6f180da45b2d3ae06d44201197df38504d7/src/lxc/confile.c#L1651

Reply:
very annoying, thanks